torch/_dynamo/codegen.py

```diff
@@ -67,7 +71,7 @@ class PyCodegen:
 
     def __init__(
         self,
-        tx=None,
+        tx: Optional["InstructionTranslator"] = None,
```

Comment:
Seems `tx` won't be None and always be set

See: https://github.com/search?q=repo%3Apytorch%2Fpytorch%20PyCodegen(&type=code

Reply:
```suggestion
        tx: "InstructionTranslator",
```

Reply:
@Skylion007 Could you please review this again? Thanks!